### PR TITLE
PIV federation SP number assigned

### PR DESCRIPTION
**Pull request by hferraio**
The number is SP 800-217.  I suggest a  title, but we can discuss if we want to change/edit title.

### CODE REVIEW

File: _FIPS201/introduction.md [hunk 1]
```diff
@@ -166,7 +166,7 @@ The requirements for the accreditation of PIV Card issuers are specified in [[SP
 The unique organizational codes for federal agencies are assigned in [[SP 800-87]](../_Appendix/references.md#ref-SP-800-87).
 The requirements for PIV Card readers are provided in [[SP 800-96]](../_Appendix/references.md#ref-SP-800-96).
 The format for encoding PIV enrollment records for import and export is specified in [[SP 800-156]](../_Appendix/references.md#ref-SP-800-156).
-The requirements for issuing derived PIV credentials are specified in [[SP 800-157]](../_Appendix/references.md#ref-SP-800-157).
+The requirements for issuing derived PIV credentials are specified in [[SP 800-157]](../_Appendix/references.md#ref-SP-800-157). Guidelines for the use of federations with PIV Credentials will be specified in [[SP 800-217]](../_Appendix/references.md#ref-SP-800-217).
```
Comment (jricher) [inline _FIPS201/introduction.md:169]:
```suggestion
The requirements for issuing derived PIV credentials are specified in [[SP 800-157]](../_Appendix/references.md#ref-SP-800-157). Guidelines for the use of federation with PIV Credentials will be specified in [[SP 800-217]](../_Appendix/references.md#ref-SP-800-217).
```

File: _Appendix/references.md [hunk 2]
```diff
@@ -385,3 +385,8 @@ Available at <https://docs.oasis-open.org/security/saml/v2.0/saml-authn-context-
 *Guidelines for Derived Personal Identity Verification (PIV) Credentials.*
 (National Institute of Standards and Technology, Gaithersburg, MD), NIST Special Publication (SP) 800-157 [or as amended].
 <https://doi.org/10.6028/NIST.SP.800-157>
+
+[[SP 800-271]](references.md#ref-SP-800-217){:name="ref-SP-800-217"} 
+*Guidelines for the Use of Personal Identity Verification (PIV) Credentials with Federations.*
```
Comment (jricher) [inline _Appendix/references.md:390]:
```suggestion
*Guidelines for the Use of Personal Identity Verification (PIV) Credentials with Federation.*
```

File: _Appendix/references.md [hunk 3]
```diff
@@ -385,3 +385,8 @@ Available at <https://docs.oasis-open.org/security/saml/v2.0/saml-authn-context-
 *Guidelines for Derived Personal Identity Verification (PIV) Credentials.*
 (National Institute of Standards and Technology, Gaithersburg, MD), NIST Special Publication (SP) 800-157 [or as amended].
 <https://doi.org/10.6028/NIST.SP.800-157>
+
+[[SP 800-271]](references.md#ref-SP-800-217){:name="ref-SP-800-217"}
```
Comment (jimfenton) [inline _Appendix/references.md:389]:
```suggestion
[[SP 800-217]](references.md#ref-SP-800-217){:name="ref-SP-800-217"} 
```

**Review by jimfenton — CHANGES_REQUESTED** [review]
Small suggestion on the title: maybe, *Guidelines for the Federated Use of Personal Identity Verification (PIV) Credentials*

Want to include the expected DOI URL?

File: _FIPS201/federation.md [hunk 4]
```diff
@@ -12,7 +12,7 @@ anchor: s-7
 _This section is normative._ It defines a set of mechanisms for using federation technologies to 
 interoperate with PIV and derived PIV credentials issued by other agencies.
 
-Federation protocols allow a trusted IdP to assert a cardholder's identity to an RP across a network in a secure and verifiable fashion, even if the PIV Card or derived PIV credential has been issued by another agency. The processes and requirements for federation systems are discussed in depth in [[SP 800-63C]](../_Appendix/references.md#ref-SP-800-63C). 
+Federation protocols allow a trusted IdP to assert a cardholder's identity to an RP across a network in a secure and verifiable fashion, even if the PIV Card or derived PIV credential has been issued by another agency. The processes and requirements for federation systems are discussed in depth in [[SP 800-63C]](../_Appendix/references.md#ref-SP-800-63C). Additional guidelines for the use of federations with PIV Credentials will be specified in [[SP 800-217]](../_Appendix/references.md#ref-SP-800-217).
```
Comment (regenscheid) [inline _FIPS201/federation.md:15]:
```suggestion
Federation protocols allow a trusted IdP to assert a cardholder's identity to an RP across a network in a secure and verifiable fashion, even if the PIV Card or derived PIV credential has been issued by another agency. The processes and requirements for federation systems are discussed in depth in [[SP 800-63C]](../_Appendix/references.md#ref-SP-800-63C). Additional guidelines for the use of federation with PIV Credentials will be specified in [[SP 800-217]](../_Appendix/references.md#ref-SP-800-217).
```

**Pull request merged**